data/shreyanshs31.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Shreyansh Shukla",
+  "displayStatsPublic":"true",
+  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜",
+  "type": "personal",
```

the following are no longer necessary:
```suggestion
  "bio": "Student👨🏽‍🎓 • DSA👨🏽‍💻 • Python • Java •  Learning in Public👤 • Love Open Source💜",
```